include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -621,6 +621,7 @@ typedef struct OrtMIGraphXProviderOptions {
   const char* migraphx_save_model_path;              // migraphx model path name
   int migraphx_load_compiled_model;                  // migraphx int8 cal table. Default 0 = false, noznero = true
   const char* migraphx_load_model_path;              // migraphx model path name
+  int migraphx_exhaustive_tune;                      // migraphx tuned compile  Default 0 = false, nonzero = true
```

Comment:
nit: Shall we use bool instead of int?

I understand that there is history reason (bool is available since C99 for 25 years) or for consistent style.

Reply:
Fair. I was just mirroring what we did with other enables. Sure I can change this